src/Widgets/DataCountWidget/DataCountWidgetEditingConfig.ts

```diff
@@ -23,6 +23,10 @@ provideEditingConfig(DataCountWidget, {
       description:
         'The placeholder __count__ represents the total number of items.',
     },
+    margin: {
+      title: 'Margin',
+      description: 'Outer space to next item (bottom). Default: "mb-2"',
```

Comment:
* [ ] FYI not introduced here but trying to prevent the alien invasion 👽 🛸 - How about something like "Spacing below to next element."?